sdk/migrate/Azure.ResourceManager.Migrate/src/Generated/AssessedSqlMachineData.cs

```diff
@@ -0,0 +1,256 @@
+        {
+            BiosGuid = biosGuid;
+            Fqdn = fqdn;
+            SqlInstances = sqlInstances;
+            SuitabilityDetail = suitabilityDetail;
+            SuitabilityExplanation = suitabilityExplanation;
+            RecommendedVmSize = recommendedVmSize;
+            RecommendedVmFamily = recommendedVmFamily;
+            ProductSupportStatus = productSupportStatus;
+            RecommendedVmSizeNumberOfCores = recommendedVmSizeNumberOfCores;
+            RecommendedVmSizeMegabytesOfMemory = recommendedVmSizeMegabytesOfMemory;
+            MonthlyComputeCost = monthlyComputeCost;
+            Disks = disks;
+            NetworkAdapters = networkAdapters;
+            MonthlyBandwidthCost = monthlyBandwidthCost;
+            MonthlyStorageCost = monthlyStorageCost;
+            CostComponents = costComponents;
+            SecuritySuitability = securitySuitability;
+            MigrationGuidelines = migrationGuidelines;
+            BootType = bootType;
+            OperatingSystemType = operatingSystemType;
+            OperatingSystemName = operatingSystemName;
+            OperatingSystemVersion = operatingSystemVersion;
+            OperatingSystemArchitecture = operatingSystemArchitecture;
+            CreatedOn = createdOn;
+            UpdatedOn = updatedOn;
+            DisplayName = displayName;
+            TypePropertiesType = typePropertiesType;
+            Description = description;
+            DatacenterMachineArmId = datacenterMachineArmId;
+            DatacenterManagementServerArmId = datacenterManagementServerArmId;
+            DatacenterManagementServerName = datacenterManagementServerName;
+            MegabytesOfMemory = megabytesOfMemory;
+            NumberOfCores = numberOfCores;
+            ConfidenceRatingInPercentage = confidenceRatingInPercentage;
+            PercentageCoresUtilization = percentageCoresUtilization;
+            PercentageMemoryUtilization = percentageMemoryUtilization;
+            Suitability = suitability;
+            SizingCriterion = sizingCriterion;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Gets or sets the BIOS GUID for the machine. </summary>
+        public string BiosGuid { get; }
```

Comment:
@srilipta-swain 
It would be better if you can fix them in the next version of swagger. But if the SDK release cannot catch up with the swagger release. You can use directives to change the generated codes first.